### ISSUE TITLE
Replace usage of `ɵmarkDirty`

### ISSUE THREAD

**Issue by edbzn** [rx-angular/rx-angular#1386]
# Problem Solved By The Feature

The private `ɵmarkDirty` API was removed from the Angular core and will land in v14.2.0 so we need to refactor our global strategy.

# Solution

I think we can use `ApplicationRef#tick` instead.

# Additional Context

This PR https://github.com/angular/angular/pull/46806 removes the private export.

**Comment by hoebbelsB** [comment]
interesting idea to use `ApplicationRef#tick`. But we need to make sure to coalesce it properly, otherwise we run into heavy over-renderings. Also we don't have access to AppRef inside the strategycredentials.
I would rather suggest ditching the `global` strategy and migrate it to `native`. I guess it isn't in use anyway :shrug:

**Comment by edbzn** [comment]
If we have no access to the ApplicationRef in the strategy credentials, then I agree it would be easier to drop it. Do we want to create a migration script for that? or just drop it? I'm in favor of dropping it.

**Comment by hoebbelsB** [comment]
we can also just point global to native strategy and deprecate it

**Comment by muuvmuuv** [comment]
With the latest version, I get this, which might be related:

```
./node_modules/.pnpm/@rx-angular+cdk@1.0.0-beta.3_vurzrkwbihck2hvr3ls5yaqlwm/node_modules/@rx-angular/cdk/fesm2015/cdk-render-strategies.js:148:24-34 - Error: export 'ɵmarkDirty' (imported as 'ɵmarkDirty') was not found in '@angular/core' (possible exports: ANALYZE_FOR_ENTRY_COMPONENTS, ANIMATION_MODULE_TYPE, APP_BOOTSTRAP_LISTENER, .........)
```


    "@rx-angular/cdk": "1.0.0-beta.3"
    "@angular/common": "^14.2.0",
    "@angular/core": "^14.2.0",

**Comment by twittwer** [comment]
Do somebody know of some workaround until this is fixed?
We aren't using the global strategy but cannot start anymore after the Angular 14.2 update.

**Comment by edbzn** [comment]
This has been fixed and released in `@rx-angular/cdk@1.0.0-rc.0`.
